src/System.Management.Automation/engine/parser/TypeResolver.cs

```diff
@@ -773,6 +774,7 @@ internal static class CoreTypes
                     { typeof(sbyte),                                       new[] { "sbyte" } },
                     { typeof(string),                                      new[] { "string" } },
                     { typeof(SupportsWildcardsAttribute),                  new[] { "SupportsWildcards" } },
+                    { typeof(SuppressMessageAttribute),                    new[] { "SuppressMessageAttribute" } },
```

Comment:
Given other attribute accelerators seem to deliberately drop the `attribute` suffix in the name, I'd suggest we should do the same here.

Reply:
Didn't think of that, good point. Happy to make that change if other's agree as well. At this moment, the alias name wasn't even agreed in the issue, therefore this name is just the first proposal so I'm aware I could be asked to change it to anything. People actually suggested `SuppressAnalyzer` in the issue. Let's use this PR to decide on the name.

Reply:
I'd definitely think it best to keep the same name as the actual type unless we wanted to go even shorter, with something like `[suppress]`.

Reply:
l like being a bit more specific e.g. "suppress what"  oh, `[SuppressMessage]`.  And I like it that I can mentally add `Attribute` to get the corresponding .NET type.